services/galley/test/integration/API/MLS.hs

```diff
@@ -1630,3 +1632,81 @@ testBackendRemoveProposalLocalConvRemoteLeaver = do
         for_ bobClients $ \(_, ref) ->
           WS.assertMatch_ (5 # WS.Second) wsA $
             wsAssertBackendRemoveProposal bob qcnv ref
+
+-- for_ kprefs $ \kp ->
+--   WS.assertMatch_ (5 # WS.Second) wsA $ \notification ->
+--     void $
+--       wsAssertBackendRemoveProposal (pUserId bob) conversation kp notification
```

Comment:
I suppose this has been removed in subsequent commits, but this is just a reminder in case it's not the case.